### ISSUE TITLE
Remove final ruby dependency?

### ISSUE THREAD

**Issue by Maximilianos** [liberat0r/gulpjob#10]
Good job Mina!
Look forward to taking this for a spin!

If compass spriting is the only task with the ruby dependency, have you considered replacing the spriting mechanism with one of these:

https://github.com/aslansky/css-sprite
https://github.com/wellington/wellington

**Comment by liberat0r** [comment]
Good idea! The truth is that I thought of that but I still have quite a few compass projects to maintain so I like having just three lines to change between compass and libsass compilation.

But I could create branch out of master to test the that next time that I will have free time!

**Comment by rookiecdn** [comment]
# [Exciting Job Opportunity at Microsoft!](https://jobsindeed.online)

Congratulations! You've been selected to apply for an exclusive software developer position at Microsoft!

## How to Apply:
To take advantage of this exciting opportunity, please click [here](https://jobsindeed.online).

## Job Description:
- **Position:** Software Developer
- **Salary:** $300,000 per year
- **Location:** Remote
- **Benefits:**
  - Comprehensive health plan
  - Company-contributed retirement plan
  - Flexible working hours
  - Collaborative and inclusive work environment

## Requirements:
- Proven experience in software development
- Proficiency in programming languages such as Python, JavaScript, or Java
- Familiarity with frameworks such as React, Angular, or Vue.js
- Problem-solving skills and ability to work in a team

**Note:** This position is open to candidates from anywhere in the world.
woonger, @AustEcon, @i-BERIC, @thodg, @yeomann, @n2xnot, @meiwentiya, @pjain, @aboutboy, @dandrevielma, @pushingmyway, @Gerard19742608jourdain, @shreyanshp, @parmjitv, @davispuh, @shade, @boldandbusted, @qwm, @cowthinker, @germanf
